docs/anchoring-assets/anchor-platform/architecture.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Platform Architecture"
+sidebar_position: 20
+---
+
+## Architecture Overview
+
+Before getting started with an Anchor Platform, let's get a bit familiar with the overall architecture. You will get familiar with the different servers provided by the platform, and all components that's needed to be hosted.
+
+[![anchor platform architecture](/assets/anchor-platform-architecture.png)](/assets/anchor-platform-architecture.png)
+
+### Components
+
+#### Client
+
+Client is a user-facing application (wallet), that makes requests to the system. Clients make request to the Platform component called SEP-server, using sets of standards called [SEPs][seps] (Stellar Ecosystem Proposals)
+
+#### SEP-server
+
+SEP-server is a client-facing server (thus it needs to be accessible from the outside network). SEP server processes user requests and communicates with database, storing necessary user input. Note, that SEP server will never store any sensitive information, such as KYC (PII), in the database.
+
+#### Database
+
+Platform uses SQL database (Postgres) to store Stellar events and entities. Its primarily use is to store SEP (SEP-24/SEP-31) transactions.
+
+#### Platform Server
+
+Platform server is an internal component. It should be hosted in the private network and should not be accessible from the Internet. This server allows business server to communicate with other parts of the Anchor Platform (such as SEP-server) via set of [APIs][platform-api].
+
+#### Stellar Observer
+
+:::tip
+
+This component is optional, and may be replaced by your own implementation.
+
+:::
+
+Stellar observer tracks Stellar chain and automatically detects user payments' Stellar transactions, updating corresponding SEP transactions statuses.
+
+#### Business Server
+
+Finally, Business server is a server that you (business) must implement, in order to fully support most of the SEPs, supported by the Platform. Business server reacts to some user events (such as requesting a quote for SEP-38), and pushes updates to the Platform for user notification (for example, completing a bank transfer)
+
+## Future architecture
+
+:::caution
+
+This diagram is only served to show the planned architecture. Changes in the future versions may be different from the current diagram.
+
+:::
+
+[![future anchor platform architecture](/assets/anchor-platform-architecture-2.png)](/assets/anchor-platform-architecture-2.png)
+
+### New Components
+
+#### Event Service
+
+Event service is a new component that will push events to both user and business via webhooks. Currently, some functionality of the event service is supported via [callback APIs][callback-api]
```

Comment:
```suggestion
The event service is a new component that will push events to both the client and business via HTTP webhooks. Currently, some functionality of the event service is supported via [callback APIs][callback-api]
```

I don't think this second statement is accurate, and we might just want to remove it.

Reply:
I think I'll just move the callback API part to the SEP server description